cmd/skopeo/proxy.go

```diff
@@ -614,6 +621,51 @@ func (h *proxyHandler) GetBlob(args []interface{}) (replyBuf, error) {
 	return ret, nil
 }
 
+func (h *proxyHandler) LayerInfoJSON(args []interface{}) (replyBuf, error) {
```

Comment:
Please document the operation here. At the very least a short blurb, and the exceptional/unusual/non-obvious aspects. And there _is_ a non-obvious aspect that really should be documented: This exists because the manifest data as returned by `GetManifest` is not sufficient to correctly call `GetBlob` against c/storage sources.